### PR TITLE
added segmentation model

### PR DESCRIPTION
**Pull request by leoarc**
## Description
Added YNet model

## Motivation and Context
Expand model set

## Types of changes
**What types of changes does your code introduce? Put an `x` in all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
**Go over all the following points, and put an `x` in all the boxes that apply.
**(If you're unsure about any of these, don't hesitate to ask. We're here to help!)
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: Segmentation Sample Models/YNet/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# YNet
+## Dataset 
+The dataset used is available in [this](https://www.bcsc-research.org/data/variables/) post.
+A total of 87 pathologists diagnosed a randomly assigned subset of 60 slides into four diagnostic categories (benign, atypia, ductal carcinomain situ, and invasive cancer),producing an average of 22 diagnostic labels per case.  The average size of these ROIs is10,000×12,000. We use these 200 ROIs for our data set.
```
Comment (Insiyaa) [inline Segmentation Sample Models/YNet/README.md]: Highlight the categories, that is don't put in parenthesis as it is a required field while uploading. Also, mention the magnification of samples.

**Review by Insiyaa — CHANGES_REQUESTED** [review]
Add about image preprocessing, I really like the format you used [here](https://github.com/camicroscope/tfjs-models/tree/master/Classification%20Sample%20Models/ICIAR18_data_IRRCNN_model).

**Comment by Insiyaa** [comment]
Great! Please add a link to the model in the readme of the repository.

**Comment by leoarc** [comment]
Sorry for not including everything in one go. Please let me know if I missed something again.

**Review by birm — APPROVED** [review]
I think this is a good addition. Of course, I'll wait for @Insiyaa to approve before merging, but I do have a light suggestion; the ipnb file committed looks like it wasn't run. Does it make sense to commit a version which was run and has output?

**Comment by leoarc** [comment]
I have added the executed file except the train cell as that would take very long.Is the ok ?

**Review by Insiyaa — APPROVED** [review]
Looks good!

**Pull request merged**